package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "updog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "test-glob.js",
+  "devDependencies": {
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.3",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "prettier": "2.5.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/seanhogunkim/Updog.git"
```

Comment:
refactor to main repo links? similar question on lines 26 and 28

Reply:
Tried removing first